docs/resources/vod_transcoding_template_group.md

```diff
@@ -0,0 +1,146 @@
+  provider-level region will be used. Changing this creates a new resource.
+
+* `name` - (Required, String) Specifies the name of the template group. The value can be a string of 1 to 128
+  characters that can consist of letters, digits and underscores (_).
+
+* `common` - (Required, List) Specifies the common configurations of the template group.
+  The [object](#common_object) structure is documented below.
+
+* `quality_info`- (Required, List) Specifies the quality info list of the template group.
+  The [object](#quality_info_object) structure is documented below.
+
+* `description` - (Optional, String) Specifies the description of the template group.
+
+* `auto_encrypt` - (Optional, Bool) Specifies whether to automatically encrypt. Before enabling, you need to configure
+  the HLS encryption key URL. When `auto_encrypt` is **true**, the `output_format` must be **HLS**.
+  Defaults to: **false**.
+
+* `is_default` - (Optional, Bool) Specifies whether to use this group as default group. Defaults to: **false**.
+
+* `watermark_template_ids` - (Optional, List) Specifies the list of the watermark template IDs.
+
+<a name="common_object"></a>
+The `common` block supports:
+
+* `low_bitrate_hd` - (Optional, Bool) Specifies whether to enable low bitrate HD. Defaults to: **false**.
+
+* `audio_codec` - (Optional, String) Specifies the audio codec. The value can be: **AAC** and **HEAAC1**.
+  Defaults to: **AAC**.
+
+* `video_codec` - (Optional, String) Specifies the video codec. The value can be: **H264** and **H265**.
+  Defaults to: **H264**.
+
+* `hls_segment_duration` - (Optional, Int) Specifies the HLS segment duration. The value can be: **2**, **3**, **5**
+  and **10**. Defaults to: **5**. This parameter is used only when `output_format` is set to **HLS** or **DASH_HLS**.
+
+<a name="quality_info_object"></a>
+The `quality_info` block supports:
+
+* `output_format` - (Required, String) Specifies the output format. The value can be: **HLS**, **MP4**, **DASH**,
+  **DASH_HLS**, **MP3** and **ADTS**.
+
+* `video` - (Optional, List) Specifies the video configurations.
+  The [object](#video_object) structure is documented below.
+
+* `audio` - (Optional, List) Specifies the audio configurations.
+  The [object](#audio_object) structure is documented below.
+
+<a name="video_object"></a>
+The `video` block supports:
+
+* `quality` - (Required, String) Specifies the video quality.
+  The value can be: **4K**, **2K**, **FHD**, **SD**, **LD** and **HD**.
+
+* `width` - (Optional, Int) Specifies the video width. The value can be **0** or range from **128** to **3840**.
+  Defaults to: **0**. If set to **0**, the system will automatically adjust the `width` according to the `height`.
+
+* `height` - (Optional, Int) Specifies the video height. The value can be **0** or range from **128** to **2160**.
+  Defaults to: **0**. If set to **0**, the system will automatically adjust the `height` according to the `width`.
```

Comment:
can we set both width and height to 0?

Reply:
both setting to 0 works on console
so I don't think we should add limit here